### ISSUE TITLE
deploy-service scs fails 

### ISSUE THREAD

**Issue by PiNh00fD** [cloudfoundry-incubator/cfdev#132]
Hi,
Installed pcfdev-v1.2.0-darwin.tgz on MacBook Pro with 16GB ram (same error occurs on Linux Bionic Beaver with 16G ram, so Tried multiple OS-es). works as expecte, but when I deploy service scs, the deployment fails. Added the last part of the deployment log, since I have the feeling that the phrase "Service broker error: Could not find an available leader" has something to do with it. I have the same error when I try to bind the "spring music test app" to mysql service.
(Service broker error: Could not find an available leader from [https://10.144.0.8:8443]). Below the deploy-service scs log.

Organization 'p-spring-cloud-services' not found.
Space 'instances' not found.
Space 'p-spring-cloud-services' not found.
App spring-cloud-broker-venerable not found
App spring-cloud-broker not found
App spring-cloud-broker not found
App spring-cloud-broker-worker not found
Service instance spring-cloud-broker-db not found
Unexpected Response
Response code: 502
CC code: 0
CC error code:
Request ID: 358f67d4-112b-499f-7b5b-d5b9a7631289::082b9a30-0afa-47f2-93ea-36d3da80ca33
Description: {
"description": "Service broker error: Could not find an available leader from [https://10.144.0.13:8443]",
"error_code": "CF-ServiceBrokerBadResponse",
"code": 10001,
"http": {
"uri": "http://10.144.0.7:8080/v2/service_instances/828aa933-a3f4-48bd-b254-367ca165374a/service_bindings/4ce8d30a-ef09-433a-b51a-3c120b7b7f11",
"method": "PUT",
"status": 500

**Comment by aemengo** [comment]
Could you please post you CF CLI version?

```
$ cf --version
...
```

**Comment by PiNh00fD** [comment]
> Could you please post your CF CLI version?


>

> ```

> $ cf --version

> ...
cf version
6.51.0+2acd15650.2020-04-07
�

**Comment by PiNh00fD** [comment]
Hi,
cf version is:
cf version 6.51.0+2acd15650.2020-04-07

**Comment by aemengo** [comment]
As was mentioned in https://github.com/pivotal-cf/pcfdev/issues/393#issuecomment-652761586:

This is caused by the expiration of internal certificates used in the Spring Cloud Services execution path.

```sh
# /var/vcap/sys/log/broker/broker.log
[on-demand-service-broker] [f347182b-3834-4f1d-9648-d92ebbe8fb51] 2020/06/30 22:47:58.061686 service adapter will delete binding with ID b3756eda-c00f-42c2-b9fa-dbb78fd06ddd for instance 0b252b61-0831-4cf3-a920-bd68dc012a20
[on-demand-service-broker] [f347182b-3834-4f1d-9648-d92ebbe8fb51] 2020/06/30 22:47:58.088705 external service adapter exited with 1 at /var/vcap/packages/odb-service-adapter/bin/service-adapter:
stdout:
    'Could not find an available leader from [https://10.144.0.13:8443]',
stderr: '
    [odb-sdk] handling delete-binding
    [dedicated-mysql-adapter] 2020/06/30 22:47:58 Using agent deleter
    [dedicated-mysql-adapter] 2020/06/30 22:47:58 Couldn't get read only information about URL https://10.144.0.13:8443, error: &errors.errorString{s:"GET https://10.144.0.13:8443/status failed: Get https://10.144.0.13:8443/status: x509: certificate has expired or is not yet valid"}
    [odb-sdk] Could not find an available leader from [https://10.144.0.13:8443]
    '
```
To rectify, I just released updates for the CF Dev product that will have newer certificates. This means that the problem is abated but, of course, not resolved forever.

**New Release**
- **v0.0.18** (plugin) - https://plugins.cloudfoundry.org/#cfdev
- **v1.3.0** (Service Assets) - https://network.pivotal.io/products/pcfdev/#/releases/685614

Thank you for raising this issue.

**Comment by PiNh00fD** [comment]
Hi @aemengo,
Thnx!
I can confirm that the install service scs works and I'm able to bind the spring-music app to the mysql service.
Thnx again for the support.
